test/e2e/operator_test.go

```diff
@@ -869,6 +871,158 @@ func TestRouteAdmissionPolicy(t *testing.T) {
 	}
 }
 
+func TestSyslog(t *testing.T) {
+	// Set up rsyslog.
+	syslogPod := &corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: "openshift-ingress",
+			Name:      "syslog",
+		},
+		Spec: corev1.PodSpec{
+			Containers: []corev1.Container{
+				{
+					Name:  "syslog",
+					Image: "openshift/origin-haproxy-router:v4.0",
```

Comment:
Recycle the operand image used by the payload being tested?

Reply:
You mean first get the ingress controller deployment, look up the image that it is using, and use it here?  All right, makes sense.

Reply:
Yeah, although it might actually be in cluster config API somewhere

Reply:
Done.